pkg/ccl/changefeedccl/changefeed_dist.go

```diff
@@ -188,7 +188,8 @@ func distChangefeedFlow(
 
 	// Copy the evalCtx, as dsp.Run() might change it.
 	evalCtxCopy := *evalCtx
-	dsp.Run(planCtx, noTxn, &p, recv, &evalCtxCopy, finishedSetupFn)
+	cleanup := dsp.Run(planCtx, noTxn, &p, recv, &evalCtxCopy, finishedSetupFn)
```

Comment:
`dsp.Run(planCtx, noTxn, &p, recv, &evalCtxCopy, finishedSetupFn)()` is even simpler

Reply:
Yeah, I wasn't sure if this is "stylistically acceptable" as I put it in the comment above (because it kind of hides that a function is returned and executed right away, at least to me it seems more complicated to read such code), but it does seem nicer overall. And since you pointed it out as well, let's go with calling the function right away.